src/util/date-helper.ts

```diff
@@ -1,55 +1,43 @@
+const MONTHS_WITH_31_DAYS = ["1", "3", "5", "7", "8", "10", "12"];
+const MONTHS_WITH_30_DAYS = ["4", "6", "9", "11"];
+
 export namespace DateHelper {
+    /**
+     * Ensures that the given value is transformed into a logical day number (i.e. 1 - 31 days)
+     * @param day input day number
+     * @param month input month number
+     * @param month input year number
+     */
     export const clampDay = (
         day: string,
         month: string,
         year: string
     ): string => {
-        const dayNumber = parseInt(day);
```

Comment:
the `parseInt` logic is still applicable here because month may get padded to "01" and fails the includes check